joern-cli/frontends/c2cpg/src/main/scala/io/joern/c2cpg/astcreation/AstForExpressionsCreator.scala

```diff
@@ -78,7 +80,10 @@ trait AstForExpressionsCreator(implicit withSchemaValidation: ValidationMode) {
 
   private def astForCppCallExpression(call: ICPPASTFunctionCallExpression): Ast = {
     val functionNameExpr = call.getFunctionNameExpression
-    val typ              = functionNameExpr.getExpressionType
+    val typ = Try(functionNameExpr.getExpressionType) match {
+      case Failure(_)   => return astForCppCallExpressionUntyped(call)
```

Comment:
a shortpath `return statement` isn't considered 'nice' scala code and may even result in a new compiler warning - consider rewriting into something like 

```
Try(functionNameExpr.getExpressionType).map { typ => 
...
}.getOrElse(astForCppCallExpressionUntyped(call))
```